recipes/kneebow/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "kneebow" %}
+{% set version = "1.0.2" %}
+{% set python_min = "3.8" %}  # Updated to reflect conda-forge minimum Python version if needed.
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/kneebow-{{ version }}.tar.gz
+  sha256: 606470056c279cc9ec8655af634787b42149ba3d36d9f71a8524007eef55e618
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+
+  run:
+    - python >={{ python_min }}
+    - numpy >=1.20
```

Comment:
The upstream `setup.py` does not constrain the versions of `numpy, `matplotlib` or `scikit-learn`. What's the reasoning behind setting the versions you've chosen here?

Reply:
I removed the specified versions.